### PR TITLE
Forge tests

### PR DESCRIPTION
**Pull request by drgorillamd**

### CODE REVIEW

**Comment by mejango** [comment]
image
we can get rid of Test contract now right?

File: contracts/JBProjectHandles.sol [hunk 1]
```diff
@@ -1,204 +1,269 @@
...
+
+    @dev 
+    See https://eips.ethereum.org/EIPS/eip-137.
+
+    @param _ensNameParts The parts of an ENS name to hash.
+
+    @return namehash The namehash for an ensName.
+  */
+  function _namehash(string[] memory _ensNameParts) internal pure returns (bytes32 namehash) {
+    namehash = 0x0000000000000000000000000000000000000000000000000000000000000000;
```
Comment (xBA5ED) [inline contracts/JBProjectHandles.sol:229]: nit: `namehash` is already the 0 value

**Review by xBA5ED — COMMENTED** [review]
lgtm

File: contracts/JBProjectHandles.sol (hunk 1 above)
Comment (mejango) [reply contracts/JBProjectHandles.sol]: nice, done.

File: test/JBProjectHandles.t.sol [hunk 2]
```diff
@@ -55,15 +63,163 @@ contract ContractTest is Test {
...
+        assertEq(projectHandle.handleOf(_projectId),string(abi.encodePacked(_name, '.', _subdomain)) );
+    }
+
+    function assertEq(ENSName memory _first, ENSName memory _second) internal {
+        assertEq(keccak256(bytes(_first.name)), keccak256(bytes(_second.name)));
+        assertEq(keccak256(bytes(_first.subdomain)), keccak256(bytes(_second.subdomain)));
+    }
+
+    function _namehash(ENSName memory _ensName) internal pure returns (bytes32 namehash) {
+            namehash = 0x0000000000000000000000000000000000000000000000000000000000000000;
```
Comment (viraj124) [inline test/JBProjectHandles.t.sol]: nit -> pretty minor (this can be removed)

**Review by viraj124 — APPROVED** [review]
LGTM

**Pull request merged**